doc/classes/FoldableContainer.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FoldableContainer" inherits="Container" keywords="expandable, collapsible, collapse" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A container that can be expanded/collapsed.
+	</brief_description>
+	<description>
+		A container that can be expanded/collapsed, with a title that can contain buttons.
+		The title can be positioned at the top or bottom of the container.
+		The container can be expanded or collapsed by clicking the title or by pressing [code]ui_accept[/code] when focused.
+		Child control nodes are hidden when the container is collapsed. Ignores non-control children.
+		Can allow grouping with other [FoldableContainer]s, check [member foldable_group] and [FoldableGroup].
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="add_title_bar_control">
+			<return type="void" />
+			<param index="0" name="control" type="Control" />
+			<description>
+				Adds a [Control] that will be placed next to the container's title, obscuring the clickable area. Prime usage is adding [Button] nodes, but it can be any [Control].
+				The control will be added as a child of this container and removed from previous parent if necessary. The controls will be placed aligned to the right, with the first added control being the leftmost one.
+			</description>
+		</method>
+		<method name="expand">
+			<return type="void" />
+			<description>
+				Expands the container and emits [signal folding_changed].
+			</description>
+		</method>
+		<method name="fold">
+			<return type="void" />
+			<description>
+				Folds the container and emits [signal folding_changed].
+			</description>
+		</method>
+		<method name="remove_title_bar_control">
+			<return type="void" />
+			<param index="0" name="control" type="Control" />
+			<description>
+				Removes a [Control] added with [method add_title_bar_control].
```

Comment:
Add a hint here that you need to manually free nodes after removing them from the title bar.